Interview_Prep_App/native_kotlin/clean-architecture/build.gradle.kts

```diff
@@ -0,0 +1,72 @@
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+    kotlin("kapt")
+    id("com.google.dagger.hilt.android")
+    kotlin("plugin.serialization")
+    id("kotlin-parcelize")
+    id("androidx.navigation.safeargs.kotlin")
+}
+
+android {
+    compileSdk = rootProject.extra.get("compileSdkVersion") as Int
+
+    defaultConfig {
+        applicationId = "com.interviewprep.template"
```

Comment:
Please rename to something you like better, or "com.interviewPrep.kotlin_clean" lol

Reply:
Good catch, forgot about that hahaha